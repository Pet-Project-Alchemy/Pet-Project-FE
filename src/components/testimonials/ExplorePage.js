import React from 'react';
import ImageGallery from './ImageGallery';
import Testimonials from './Testimonials';

export default function Explore() {
  return (
    <>
      <Testimonials />
      <ImageGallery />
    </>
  );
}
