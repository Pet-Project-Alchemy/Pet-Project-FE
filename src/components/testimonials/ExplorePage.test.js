import React from 'react';
import { shallow } from 'enzyme';
import ExplorePage from './ExplorePage';

describe('ExplorePage component', () => {
  it('renders ExplorePage', () => {
    const wrapper = shallow(<ExplorePage />);
    expect(wrapper).toMatchSnapshot();
  });
});
