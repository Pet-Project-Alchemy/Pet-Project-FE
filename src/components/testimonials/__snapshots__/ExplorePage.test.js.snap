// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExplorePage component renders ExplorePage 1`] = `
<Fragment>
  <div
    className="explore-page"
  >
    <Testimonials />
    <ImageGallery />
  </div>
</Fragment>
`;
